src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -341,8 +341,10 @@ private void updateItemTransactionOnRequestChange(RequestDTO requestDTO, InnReac
   private void updatePatronTransactionOnRequestChange(RequestDTO requestDTO, InnReachTransaction transaction) {
     if (requestDTO.getStatus() == CLOSED_CANCELLED) {
       log.info("Updating patron hold transaction {} on cancellation of a request {}", transaction.getId(), requestDTO.getId());
```

Comment:
move the log into condition, since we don't update anything if condition is not met